pgamit/pyETM.py

```diff
@@ -183,28 +184,32 @@ def __init__(self, m_type, **kwargs):
                 elif isinstance(arg, float):
                     self.relaxation = np.array(arg)
                 else:
-                    raise pyETMException_Model('\'relaxation\' must be list, numpy.ndarray, or float')
+                    raise pyETMException_Model(
+                        '\'relaxation\' must be list, numpy.ndarray, or float')
```

Comment:
...I think from here to line 224, we probably don't need the python string escape characters (`\`) that are used. If we want quoted quotes, we can just switch from double to single without breaking the string:

```python
                        "'relaxation' must be list, numpy.ndarray, or float")
```